### PR TITLE
Merge of 2.2.0 branch prior to release

### PR DESCRIPTION
**Pull request by seangates**
## Description
- Textbox: added textbox-keydown event (#626)
- Infotip: disable attribute (#611)
- Textbox: Event when floating label is present and component is updated; (#610)
- Combobox: new editable combobox (#566)
- Fix issue with not being able to close tourtip (#634)
- Check for option text; (#635)
- Fixes the initial currentValue; (#637)
- Combobox: chrome autofill and name passthrough (#647)
- Upgraded Skin for Edge/IE textbox issue (#650)

### CODE REVIEW

File: src/components/ebay-combobox/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# ebay-combobox
+
+_**For Desktop only**_
```
Comment (ianmcburnie) [inline src/components/ebay-combobox/README.md]: Very minor, but I think it's no longer desktop only.

File: src/components/ebay-combobox/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+The `<ebay-combobox>` is used to create a custom-designed form element which matches much of the functionality of a native `<select>` menu. It is a combination of a text `<input>`, and a listbox (`aria-role="listbox"`). It supports both written text by the user, as well as text selected from the listbox options.
```
Comment (ianmcburnie) [inline src/components/ebay-combobox/README.md]: This needs updating before release.

File: src/components/ebay-combobox/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <ebay-combobox-option text="Option 3"/>
+</ebay-combobox>
+```
+
+### ebay-combobox Attributes
+
+Name | Required | Type | Stateful | Description
+--- | --- | --- | --- | ---
+`name` | Yes | String | No | used for the `name` attribute of the native `<select>`
+`borderless` | No | Boolean | No | whether button has borders
```
Comment (ianmcburnie) [inline src/components/ebay-combobox/README.md]: button?

Comment (seangates) [reply src/components/ebay-combobox/README.md]: Good catch. Will update.

File: src/components/ebay-combobox/README.md (hunk 1 above)
Comment (seangates) [reply src/components/ebay-combobox/README.md]: Let me test this quickly before I remove this.

File: src/components/ebay-combobox/README.md (hunk 2 above)
Comment (seangates) [reply src/components/ebay-combobox/README.md]: Ah, yeah. Let me fix this.

File: src/components/ebay-combobox/README.md (hunk 1 above)
Comment (seangates) [reply src/components/ebay-combobox/README.md]: Appears to work well on mobile (iOS), but long option lists might pose an issue there. We could address this through JS at a later time if it becomes an issue, but presently is low priority in my book.

Comment (seangates) [reply src/components/ebay-combobox/README.md]: Hence, removing this statement.

Comment (ianmcburnie) [reply src/components/ebay-combobox/README.md]: Agreed!

**Pull request merged**